tests/packagedcode/data/pypi/unpacked_sdist/prefer-egg-info-pkg-info/celery-expected.json

```
@@ -0,0 +1,8105 @@
+              "scope": "install",
+              "is_runtime": true,
+              "is_optional": false,
+              "is_resolved": false,
+              "resolved_package": {},
+              "extra_data": {}
+            },
+            {
+              "purl": "pkg:pypi/click",
+              "extracted_requirement": "click<9.0,>=8.0.3",
+              "scope": "install",
+              "is_runtime": true,
+              "is_optional": false,
+              "is_resolved": false,
+              "resolved_package": {},
+              "extra_data": {}
+            },
+            {
+              "purl": "pkg:pypi/click-didyoumean",
+              "extracted_requirement": "click-didyoumean>=0.0.3",
+              "scope": "install",
+              "is_runtime": true,
+              "is_optional": false,
+              "is_resolved": false,
+              "resolved_package": {},
+              "extra_data": {}
+            },
+            {
+              "purl": "pkg:pypi/click-repl",
+              "extracted_requirement": "click-repl>=0.2.0",
+              "scope": "install",
+              "is_runtime": true,
+              "is_optional": false,
+              "is_resolved": false,
+              "resolved_package": {},
+              "extra_data": {}
+            },
+            {
+              "purl": "pkg:pypi/click-plugins",
+              "extracted_requirement": "click-plugins>=1.1.1",
+              "scope": "install",
+              "is_runtime": true,
+              "is_optional": false,
+              "is_resolved": false,
+              "resolved_package": {},
+              "extra_data": {}
+            },
+            {
+              "purl": "pkg:pypi/importlib-metadata",
+              "extracted_requirement": "importlib-metadata>=1.4.0; python_version < \"3.8\"",
+              "scope": "install",
+              "is_runtime": true,
+              "is_optional": false,
+              "is_resolved": false,
+              "resolved_package": {},
+              "extra_data": {}
+            },
+            {
+              "purl": "pkg:pypi/pyarango",
+              "extracted_requirement": "pyArango>=1.3.2; extra == \"arangodb\"",
+              "scope": "arangodb",
+              "is_runtime": true,
+              "is_optional": false,
+              "is_resolved": false,
+              "resolved_package": {},
+              "extra_data": {}
+            },
+            {
+              "purl": "pkg:pypi/cryptography",
+              "extracted_requirement": "cryptography; extra == \"auth\"",
+              "scope": "auth",
+              "is_runtime": true,
+              "is_optional": false,
+              "is_resolved": false,
+              "resolved_package": {},
+              "extra_data": {}
+            },
+            {
+              "purl": "pkg:pypi/azure-storage-blob@12.9.0",
+              "extracted_requirement": "azure-storage-blob==12.9.0; extra == \"azureblockblob\"",
+              "scope": "azurebl
```

We should have instead:
```suggestion
              "extracted_requirement": "couchbase>=3.0.0",
```